### PR TITLE
chore: add store_id as span attribute for check

### PR DESCRIPTION
**Pull request by miparnisari**
This is so that we can make more specific queries for traces when debugging a particular store's performance.

### CODE REVIEW

File: internal/graph/cached_resolver.go [hunk 1]
```diff
@@ -139,10 +140,13 @@ func (c *CachedCheckResolver) ResolveCheck(
 	ctx context.Context,
 	req *ResolveCheckRequest,
 ) (*ResolveCheckResponse, error) {
-	ctx, span := tracer.Start(ctx, "ResolveCheck")
+	ctx, span := tracer.Start(ctx, "ResolveCheck", trace.WithAttributes(
```
Comment (miparnisari) [inline internal/graph/cached_resolver.go:143]: I feel inclined to change the span name to `CachedCheckResolver.ResolveCheck`, `LocalChecker.ResolveCheck`, etc. WDYT?

Comment (sergiught) [reply internal/graph/cached_resolver.go:143]: Sounds good to me 👍🏻

File: internal/graph/cycle_check_resolver.go [hunk 2]
```diff
@@ -29,10 +30,12 @@ func (c *CycleDetectionCheckResolver) ResolveCheck(
 	ctx context.Context,
 	req *ResolveCheckRequest,
 ) (*ResolveCheckResponse, error) {
-	ctx, span := tracer.Start(ctx, "ResolveCheck")
-	defer span.End()
-	span.SetAttributes(attribute.String("resolver_type", "CycleDetectionCheckResolver"))
-	span.SetAttributes(attribute.String("tuple_key", req.GetTupleKey().String()))
+	ctx, span := tracer.Start(ctx, "ResolveCheck", trace.WithAttributes(
```
Comment (sergiught) [inline internal/graph/cycle_check_resolver.go:33]: Are we missing a `defer span.End()` by any chance?

Comment (miparnisari) [reply internal/graph/cycle_check_resolver.go:33]: good catch :D

**Review by jon-whit — APPROVED** [review]

File: internal/graph/cached_resolver.go (hunk 1 above)
Comment (miparnisari) [reply internal/graph/cached_resolver.go:143]: I'll send a different PR.

**Pull request merged**